code/modules/mob/hear_say.dm

```diff
@@ -61,12 +61,12 @@
 		if(get_preference_value(/datum/client_preference/ghost_ears) == GLOB.PREF_ALL_SPEECH && (can_see(speaker)))
 			message = "<b>[message]</b>"
 
-	if(!can_hear(speaker) || get_sound_volume_multiplier() < 0.2)
```

Comment:
Please give me an alternative approach to fixing holopads that does not require adding an extra argument to hear_say and can_hear that bypasses your range checks.

Also, why are you doing this in hear_say and not in say.dm files where hear_say is called? This would avoid the problem entirely, as custom audio insulation behavior can be done for different mobs.